app/src/main/java/doit/study/droid/topic/TopicModelView.kt

```diff
@@ -0,0 +1,85 @@
+package doit.study.droid.topic
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import doit.study.droid.data.local.QuizDatabase
+import doit.study.droid.data.local.entity.Tag
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+import timber.log.Timber
+import javax.inject.Inject
+
+class TopicModelView @Inject constructor(
+        private val quizDatabase: QuizDatabase
+) : ViewModel() {
+    private val _items = MutableLiveData<List<TopicView>>().apply { value = emptyList() }
+    val items: LiveData<List<TopicView>> = _items
+
+    init {
+        loadTopics()
+    }
+
+    fun loadTopics(query: String = "") {
+        viewModelScope.launch {
```

Comment:
You can pass context to `launch` as a parameter instead of writing `withContext` inside